crates/bevy_window/src/window.rs

```diff
@@ -584,6 +584,22 @@ impl Window {
     pub fn raw_window_handle(&self) -> RawWindowHandleWrapper {
         self.raw_window_handle.clone()
     }
+
+    /// The "html canvas" element selector, if it was configured for this window.
+    ///
+    /// This value has no effect on non-web platforms.
+    #[inline]
+    pub fn canvas(&self) -> Option<&str> {
+        self.canvas.as_deref()
+    }
+
+    /// Whether or not to fit the canvas element's size to its parent element's size.
+    /// 
```

Comment:
```suggestion
    ///
```

Reply:
Looks like I inadvertently fixed this in recent commit.